packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine.sol

```diff
@@ -208,7 +211,8 @@ contract GenArt721CoreV3_Engine is
     address public minterContract;
 
     /// starting (initial) project ID on this contract
-    uint256 public immutable startingProjectId;
+    /// set on initialization
```

Comment:
nit: in general, I think it would be nice to comment on all variables like this that are immutable in practice, but aren't immutable in the compiler due to use of ERC-1167 minimal proxy pattern.

Reply:
🙏 Added additional clarification here and to the other affected variables